core-web/libs/utils-testing/src/lib/dot-workflows-actions.mock.ts

```diff
@@ -82,3 +83,243 @@ export const mockPublishAction = {
     showOn: ['LOCKED'],
     actionInputs: []
 };
+
+/**
+ * Mock of a ContentType with single workflow actions
+ */
+export const MOCK_SINGLE_WORKFLOW_ACTIONS: DotCMSWorkflowActions[] = [
+    {
+        scheme: {
+            id: 'd61a59e1-a49c-46f2-a929-db2b4bfa88b2',
+            archived: false,
+            name: 'System Workflow',
+            creationDate: new Date(1731532550440),
+            defaultScheme: false,
+            description: '',
+            entryActionId: null,
+            mandatory: false,
+            modDate: new Date(1730906400422),
+            system: true,
+            variableName: 'SystemWorkflow'
+        },
+        action: {
+            id: 'ceca71a0-deee-4999-bd47-b01baa1bcfc8',
+            name: 'Save',
+            icon: 'workflowIcon',
+            showOn: ['EDITING', 'PUBLISHED', 'UNPUBLISHED', 'LOCKED', 'NEW'],
+            assignable: false,
+            commentable: false,
+            condition: '',
+            hasArchiveActionlet: false,
+            hasCommentActionlet: false,
+            hasDeleteActionlet: false,
+            hasDestroyActionlet: false,
+            hasMoveActionletActionlet: false,
+            hasMoveActionletHasPathActionlet: false,
+            hasOnlyBatchActionlet: false,
+            hasPublishActionlet: false,
+            hasPushPublishActionlet: false,
+            hasResetActionlet: false,
+            hasSaveActionlet: false,
+            hasUnarchiveActionlet: false,
+            hasUnpublishActionlet: false,
```

Comment:
This response makes really hard to use for consumers. I will recommend revisiting the whole response to provide a better DX.

cc: @dsilvam

Reply:
@jdotcms

Reply:
This comes from `${this.BASE_URL}/defaultactions/contenttype/${contentTypeName}`